nautobot_ssot/templates/nautobot_ssot/sync_detail.html

```diff
@@ -119,6 +119,41 @@
                         </td>
                     </tr>
                 </table>
+                <div class="panel-heading">
```

Comment:
This should be a separate panel from the one above.

```suggestion
            </div>
            <div class="panel panel-default">
                <div class="panel-heading">
```